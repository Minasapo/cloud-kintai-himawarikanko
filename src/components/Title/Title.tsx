import { Typography } from "@mui/material";

export default function Title({ children }: { children: React.ReactNode }) {
  return (
    <Typography
      variant="h4"
      sx={{ pl: 1, borderBottom: "solid 5px #0FA85E", color: "#0FA85E" }}
    >
      {children}
    </Typography>
  );
}
