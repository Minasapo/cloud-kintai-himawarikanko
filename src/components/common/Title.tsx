import { Typography } from "@mui/material";

interface TitleProps {
  text: string;
}

export default function Title({ text }: TitleProps) {
  return (
    <Typography
      variant="h4"
      sx={{ pl: 1, borderBottom: "solid 5px #0FA85E", color: "#0FA85E" }}
    >
      {text}
    </Typography>
  );
}
