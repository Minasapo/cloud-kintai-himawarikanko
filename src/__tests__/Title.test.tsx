import { render } from "@testing-library/react";

import Title from "../components/Title/Title";

// スナップショットテスト
test("Title component snapshot", () => {
  const { asFragment } = render(<Title text="Title" />);
  expect(asFragment()).toMatchSnapshot();
});
