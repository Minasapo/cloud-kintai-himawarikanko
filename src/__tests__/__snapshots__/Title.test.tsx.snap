// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component snapshot 1`] = `
<DocumentFragment>
  <h4
    class="MuiTypography-root MuiTypography-h4 css-1u87xcl-MuiTypography-root"
  >
    Title
  </h4>
</DocumentFragment>
`;
